proto/protocolapis/astria/protocol/transaction/v1/action.proto

```diff
@@ -211,6 +212,35 @@ message BridgeSudoChange {
   string fee_asset = 4;
 }
 
+message BridgeTransfer {
+  // the address of the bridge account to transfer to
+  astria.primitive.v1.Address to = 1;
+  // the amount to transfer
+  astria.primitive.v1.Uint128 amount = 2;
+  // the asset used to pay the transaction fee
+  string fee_asset = 3;
+  // the address on the destination chain which
+  // will receive the bridged funds
+  string destination_chain_address = 4;
+
+  // The memo field can be used to provide unique identifying additional
+  // information about the bridge unlock transaction.
+  string memo = 5;
```

Comment:
Do we need a memo here?

Reply:
not necessarily, do you see any future use for it? i left it because BridgeUnlock has it, but don't think BridgeUnlock necessarily needs it either. removed for now